src/main/resources/avro/referencemethods.avdl

```diff
@@ -261,17 +260,89 @@ ListReferenceBasesResponse getReferenceBases(
   /** Additional request parameters to restrict the query. */
   ListReferenceBasesRequest request) throws GAException;
 
-/****************  /sequence/{id}  *******************/
+/******************  /sequences/search  *********************/
+/**
+This request maps to the body of `POST /sequences/search` as JSON.
+*/
+record SearchSequencesRequest {
+  /**
+  If not null, return only `Sequence`s that appear in the indicated
+  `ReferenceSet`, or any included `ReferenceSet`s.
+  */
+  union { null, string } referenceSetId = null;
+  
+  /**
+  If not null, return only `Sequence`s that are novel in the indicated
+  `VariantSet`.
+  */
```

Comment:
Yes, @fnothaft has the right idea. Clearly this is unclear; I will change it to "`Sequence`s that are part of the indicated `VariantSet`".